### ISSUE TITLE
miou50%，macc50%

### ISSUE THREAD

**Issue by Saillxl** [wl-zhao/VPD#62]
Excellent work, I used the segmentation code to train my binary classification dataset, but why did the loss drop to 0 after 25,000 iterations, while the average metric remained at 50%? Has anyone encountered such a situation before?

**Comment by wl-zhao** [comment]
If your dataset is too small, training the whole diffusion U-Net may suffers from overfitting.

**Comment by Saillxl** [comment]
Thank you for your prompt response. I have a training set of 120,000 images, and I have also converted the masks to binary format (0, 1).
